target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -98,6 +98,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan" "0:wan" "6@eth0"
 		;;
+	hiwifi,hc5611)
+		ucidef_add_switch "switch0" \
```

Comment:
For single-port devices you can just disable VLAN on the switch and use eth0 as LAN instead. The first group of devices in this file does exactly that.

Reply:
I delete it. But luci still have the switch entry `http://192.168.1.1/cgi-bin/luci/admin/network/switch`. How can I delete it? Thanks.

Reply:
I found it on `base system -> swconfig`. thanks.